Projects/hCore/Sources/Masking.swift

```diff
@@ -50,7 +50,7 @@ public struct Masking {
         switch type {
         case .norwegianPersonalNumber:
             let age = calculateAge(from: text) ?? 0
-            return text.count == 12 && 15 ... 130 ~= age
+            return text.count == 10 && 15 ... 130 ~= age
```

Comment:
MMDDYYXXXXX is norwegian right? --> 23115994336 is a valid NID from zignsec

Reply:
So it should be 11 not 10?

Reply:
Opps, fixed now.